### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by hicom150**
Updated to latest dependencies to fix all security issues.

### CODE REVIEW

File: src/AllureReporter.js [hunk 1]
```diff
@@ -8,11 +8,16 @@ function AllureReporter(baseReporterDecorator, config) {
     config.allureReport = config.allureReport || {};
 
     var outDir = config.allureReport.reportDir ? path.resolve(config.basePath, config.allureReport.reportDir) : undefined;
+    var useBrowserName = (config.allureReport.useBrowserName != null)
+        ? config.allureReport.useBrowserName
+        : true;
+
```
[the remark this answers is not in the document]
Comment (hicom150) [reply src/AllureReporter.js]: You are absolutely right, it's been a mistake 😅 I'll make another PR to add that feature. 👍

File: test/e2e/test-runner.js [hunk 2]
```diff
@@ -22,6 +22,14 @@ karma.start({
     allureReport: {
         reportDir: 'out'
     }
-}, function (done) {
+};
 
+var server = new Server(karmaConfig, function(exitCode) {
+    process.exit(1);
```
[the remark this answers is not in the document]
Comment (hicom150) [reply test/e2e/test-runner.js]: In this case I assumed that the idea is to check if the tests run correctly, but not if they fail or pass. This `existCode` is `1` if any test fail and `0` all test pass, so maybe I can refactor this code to better reflect the idea 😁

[the remark this answers is not in the document]
Comment (hicom150) [reply test/e2e/test-runner.js]: Fixed in https://github.com/hicom150/allure-karma/commit/1248dd681794970395ec36ea15a7aa0659def924

**Comment by just-boris** [comment]
Looks good, thank you, @hicom150 

@baev merge button does not work for me, please merge.

**Comment by baev** [comment]
@hicom150 @just-boris thanks!

**Pull request merged**